tox.ini

```diff
@@ -12,6 +12,7 @@ isolated_build = true
 skip_install =
     format,checkformatting,coverage,template: true
 setenv =
+    SQLALCHEMY_WARN_20=1
```

Comment:
Enable warnings for things which are incompatible with 2.x. This should prevent us from adding any in the mean time.

Reply:
You'll need to put this in `.cookiecutter/includes/tox/setenv`

Reply:
Gah!